src/server/game/Entities/GameObject/GameObjectData.h

```diff
@@ -839,6 +848,8 @@ struct GameObjectTemplate
     std::string StringId;
     WorldPacket QueryData[TOTAL_LOCALES];
 
+    ::DestructibleHitpoint const* DestructibleHitpoint;
```

Comment:
I would move this to `GameObjectValue` instead

Reply:
alright!

Reply:
Like this then? This might cause the console logs to be a bit more spammed now that the error message is on gameobject create instead of during load times

Reply:
You can still do validation on startup, like other gameobject_template fields that reference other tables

Reply:
should be done!